.github/ISSUE_TEMPLATE/sample_app_proposal.yaml

```
@@ -0,0 +1,50 @@
+name: Sample App Proposal
```

```suggestion
name: 🚀 Sample App Proposal
```
+ emoji with title :) align with `bug-report.yaml`

---

Qute emoji😆